ruby/ruby/Ice/Proxy.rb

```diff
@@ -12,8 +12,32 @@ def ice_staticId()
                 self::ICE_ID
             end
 
-            def checkedCast(proxy, facetOrContext=nil, context=nil)
-                ice_checkedCast(proxy, self::ICE_ID, facetOrContext, context)
+            def checkedCast(proxy, facet: nil, context: nil)
```

Comment:
Switched to keyword arguments, this means that if you were using the 2nd or 3rd args you will have to update the code to use a keyword arg, see the test updates.